### PR TITLE
feat(tutorial): complete step 1 boeblingen

### PR DESCRIPTION
**Pull request by DanielaImmel**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/components/TutorialHeader/TutorialHeader.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  SkipToContent,
+} from 'carbon-components-react/lib/components/UIShell';
+
+const TutorialHeader = () => (
+  <Header aria-label="Carbon Tutorial">
+    <SkipToContent />
+    <HeaderName element={Link} to="/" prefix="IBM">
+      Carbon Tutorial
+    </HeaderName>
+    <HeaderMenuItem element={Link} to="/repos">
```
Comment (ZrianinaMariia) [inline src/components/TutorialHeader/TutorialHeader.js]: There is a HeaderNavigation tag missing that causes css problems. Please add.

**Comment by ZrianinaMariia** [comment]
@DanielaImmel almost there! Please take a look at my review message and adjust accordingly.

**Comment by DanielaImmel** [comment]
requested changes added

**Comment by ZrianinaMariia** [comment]
@DanielaImmel Looks amazing 🏆! I've recorded your completion of step 1.

**Pull request closed without merging**